test/integration/api/klusterlet_test.go

```diff
@@ -37,6 +37,23 @@ var _ = Describe("Create Klusterlet API", func() {
 			Expect(err).NotTo(BeNil())
 		})
 	})
+
+	Context("Create with aws auth and invalid arn", func() {
+		It("should reject the klusterlet creation", func() {
+			klusterlet.Spec.RegistrationConfiguration = &operatorv1.RegistrationConfiguration{
+				RegistrationDriver: operatorv1.RegistrationDriver{
+					AuthType: "awsirsa",
+					AwsIrsa: &operatorv1.AwsIrsa{
+						ManagedClusterArn: "arn:aws:bks:us-west-2:123456789012:cluster/managed-cluster1",
+						HubClusterArn:     "arn:aws:eks:us-west-2:123456789012:cluster/hub-cluster1",
+					},
+				},
+			}
+			_, err := operatorClient.OperatorV1().Klusterlets().Create(context.TODO(), klusterlet, metav1.CreateOptions{})
+			fmt.Println(err)
+			Expect(err).NotTo(BeNil())
+		})
+	})
```

Comment:
thanks for adding the test. Would you also add a valid arn to ensure the correct format can pass the validation.

Reply:
added